app/dash/Main/index.js

```diff
@@ -22,6 +22,8 @@ class Settings extends React.Component {
   }
 
   appInfo() {
+    // TODO: move this to global passed over IPC
```

Comment:
good idea. is there a ticket for exposing globals in this way? if so, let's add it there

Reply:
I'll add it to https://linear.app/frame/issue/ENG-177/make-platform-a-globally-available-constant

Reply:
:+1: